### DISCUSSION TITLE
How to disable console output when configuring?

### DISCUSSION THREAD

**Discussion by jmigual** [microsoft/vscode-cmake-tools#3044] [Q&A] [answered]
Sometimes it annoys me that the output console gets opened whenever I configure CMake. Is there a setting to disable it?

**Comment by jmigual** [comment, marked as answer]
Nevermind I found it. There's the setting `cmake.revealLog` for that.

**Reply by bobbrow** [reply]
I believe the setting you are looking for is `cmake.revealLog`.  Set it to `"never"`.

**Reply by jmigual** [reply]
True I made a typo when copy/pasting the setting ID. I'll edit my answer. Thank you very much for your work, this extension made my life so much easier!
